src/routing.rs

```diff
@@ -507,7 +543,10 @@ impl<E> RoutingDsl for EmptyRouter<E> {}
 
 impl<E> crate::sealed::Sealed for EmptyRouter<E> {}
 
-impl<B, E> Service<Request<B>> for EmptyRouter<E> {
+impl<B, E> Service<Request<B>> for EmptyRouter<E>
+where
+    B: Send + Sync + 'static,
```

Comment:
This additional trait bound makes this a breaking change. However I doubt anyone will actually hit this in practice.